### ISSUE TITLE
Examples more friendly to share on social networks

### ISSUE THREAD

**Issue by maximlt** [holoviz-topics/examples#476]
A feature that blogs usually get for free, allowing them to share links on social networks (chats, forums, etc.) that look pretty, with an image, description, etc.

- https://github.com/wpilibsuite/sphinxext-opengraph
- https://github.com/choldgraf/sphinx-social-previews
- Sphinx meta directive https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#html-metadata
- To preview a specific page: https://socialsharepreview.com/

**Comment by jbednar** [comment]
Good use case for https://github.com/holoviz-topics/examples/issues/91 ! :-)

**Comment by maximlt** [comment]
Yep, although we'll need to make sure social platforms follow the redirects, to the page that has the opengraph/whatever metadata.
